backend/game_server/lib/cell.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Cell do
```

Comment:
I believe you should use main namespace prefix in any modules.
it's easy to imagine conflicts with some external libraries with so simple module name.
e.g. `BernacleServer.Cell`